DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/TutorialViewModelTests.cs

```diff
@@ -219,13 +219,18 @@ public void Tutorial_should_have_nextLinkViewModel()
         }
 
         [Test]
-        public void Tutorial_should_summarise_duration()
+        public void Tutorial_should_have_timeSummary()
         {
             // Given
+            const int averageTutorialDuration = 73;
+            const int timeSpent = 41;
+
             var expectedTutorialInformation = TutorialContentHelper.CreateDefaultTutorialInformation(
-                averageTutorialDuration: 73,
-                timeSpent: 41
+                timeSpent: timeSpent,
+                averageTutorialDuration: averageTutorialDuration
             );
+            // TODO: Test different customisations settings when found as part of HEEDLS-196

```

Comment:
Minor but is this comment necessary? TODO comments are very easy to forget and leave in after the ticket it refers to is done, so I'd probably leave it out 🙂

Reply:
Yeah it's not necessary, I should remember this when I get around to this ticket so I've_ removed that in 18eef65 🙂